### PR TITLE
Extend java API with vector access

### PR DESCRIPTION
**Pull request by mccullocht**
Add `Index.get(int key)` method that returns the contents of the vector on the java heap. This method will throw an
`IllegalArgumentException` if key is not found in the index.

### CODE REVIEW

File: java/cloud/unum/usearch/Index.java [hunk 1]
```diff
@@ -296,7 +307,11 @@ public Config expansion_search(long _expansion_search) {
       System.loadLibrary("usearch"); // used for tests. This library in classpath only
     } catch (UnsatisfiedLinkError e) {
       try {
-        NativeUtils.loadLibraryFromJar("/usearch/libusearch.so");
+        if (System.getProperty("os.name").equals("Mac OS X")) {
```
Comment (mccullocht) [inline java/cloud/unum/usearch/Index.java:310]: This and the gradle changes were made to make it easier to develop on mac os. I've put these in a separate commit so it should be easy to drop them if desired.

**Comment by ashvardanian** [comment]
Thanks for the PR! Merged 🤗
